tests/unit/test-cmd/test.run.js

```diff
@@ -29,6 +29,7 @@ describe('run', () => {
       artifactsDir: path.join(sourceDir, 'web-ext-artifacts'),
       sourceDir,
       noReload: true,
+      lint: false,
```

Comment:
We should probably add a new test case in this file to ensure that the CLI option --lint is passed to the extension runner.